Non-charring/PMMA/UClan/UClan_Cone_65kW.csv

```diff
@@ -1,492 +1,492 @@
-397,-0.309387207,0.690328445,
-398,-0.349365234,0.671243591,
-399,-0.329345703,0.66473465,
-400,-0.299377441,0.661481171,
-401,-0.339355469,0.651204834,
-402,-0.359375,0.642042084,
-403,-0.349365234,0.622531548,
-404,-0.359375,0.626734314,
-405,-0.419372559,0.611639938,
-406,-0.419372559,0.607548027,
-407,-0.399353027,0.594879837,
-408,-0.419372559,0.583493118,
-409,-0.379333496,0.58252388,
-410,-0.399353027,0.569843483,
-411,-0.409362793,0.564368668,
-412,-0.479309082,0.560960236,
-413,-0.489318848,0.551036263,
-414,-0.399353027,0.543383446,
-415,-0.419372559,0.530190849,
-416,-0.439331055,0.519313126,
-417,-0.449279785,0.51308979,
-418,-0.479309082,0.514933891,
-419,-0.489318848,0.511044693,
-420,-0.479309082,0.497476845,983.15
-421,-0.459289551,0.498794785,
-422,-0.449279785,0.497561569,
-423,-0.459289551,0.482871208,
-424,-0.459289551,0.483239708,
-425,-0.469299316,0.482128258,
-426,-0.459289551,0.46896225,
-427,-0.489318848,0.468621788,
-428,-0.479309082,0.450894508,
-429,-0.479309082,0.446704178,
-430,-0.489318848,0.430698013,
-431,-0.499328613,0.435219612,
-432,-0.509277344,0.431634483,
-433,-0.509277344,0.435522652,
-434,-0.499328613,0.430779572,
-435,-0.499328613,0.41833271,
-436,-0.519287109,0.420187988,
-437,-0.499328613,0.412014389,
-438,-0.489318848,0.41153286,
-439,-0.519287109,0.411093216,
-440,-0.549316406,0.398914299,
-441,-0.539306641,0.391487656,
-442,-0.539306641,0.388700295,
-443,-0.559326172,0.393900452,
-444,-0.559326172,0.380760574,
-445,-0.579284668,0.37286541,
-446,-0.589294434,0.369069634,
-447,-0.549316406,0.363381538,
-448,-0.559326172,0.3564151,
-449,-0.539306641,0.353395119,
-450,-0.61932373,0.350066414,927.65
-451,-0.609313965,0.348100014,
-452,-0.559326172,0.343324928,
-453,-0.539306641,0.340240021,
-454,-0.529296875,0.336601028,
-455,-0.539306641,0.329190712,
-456,-0.559326172,0.327814217,
-457,-0.529296875,0.323344383,
-458,-0.529296875,0.325773735,
-459,-0.589294434,0.324305267,
-460,-0.569274902,0.324946098,
-461,-0.559326172,0.31990839,
-462,-0.519287109,0.322625847,
-463,-0.539306641,0.325572357,
-464,-0.549316406,0.308622971,
-465,-0.579284668,0.297301083,
-466,-0.579284668,0.298064575,
-467,-0.579284668,0.296314964,
-468,-0.559326172,0.292155876,
-469,-0.549316406,0.28817421,
-470,-0.539306641,0.286332817,
-471,-0.579284668,0.286798248,
-472,-0.559326172,0.282708302,
-473,-0.559326172,0.278969288,
-474,-0.559326172,0.284523144,
-475,-0.559326172,0.275120869,
-476,-0.569274902,0.279334278,
-477,-0.659240723,0.277038517,
-478,-0.629272461,0.275074539,
-479,-0.599304199,0.267033577,
-480,-0.589294434,0.264100494,898.65
-481,-0.569274902,0.265189781,
-482,-0.549316406,0.263938999,
-483,-0.559326172,0.267049675,
-484,-0.569274902,0.261428242,
-485,-0.559326172,0.258774681,
-486,-0.569274902,0.256856747,
-487,-0.549316406,0.252785568,
-488,-0.549316406,0.255033016,
-489,-0.539306641,0.255130806,
+Time,Sample Mass,HRR,Back Surface Temperature,
+[s],[g],[kW m-2],[K],
+0,71.11999512,1.61E-07,323.65,
```

Comment:
Why is there only one `Back Surface Temperature` here?  If it is constant in time, then it will be much easier to process these files if the column is filled.   Thanks